### PR TITLE
Changes for feedback. Rename Administer Unfreeze -> Unfreeze Business…

### PR DESCRIPTION
**Pull request by seeker25**
…, Administer Freeze -> Freeze Business.

*Issue #:* /bcgov/entity#13348

*Description of changes:*
Small renaming of fields, making the text field required.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: tests/unit/FilingHistoryList1.spec.ts [hunk 1]
```diff
@@ -430,7 +430,7 @@ describe('Filing History List - misc functionality', () => {
     expect(wrapper.find('.limited-restoration-period').text())
       .toContain('The Company BC1234567 LTD. was successfully')
     expect(wrapper.find('.limited-restoration-period').text())
-      .toContain('restored and active until Apr 1, 2021')
+      .toContain('restored and is active until Apr 1, 2021')
```
Comment (jonathan-longe) [inline tests/unit/FilingHistoryList1.spec.ts:433]: Thanks for fixing this!   I should have updated the test on my last PR.

**Review by jonathan-longe — APPROVED** [review]

File: src/enums/filingNames.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 export enum FilingNames {
-  ADMIN_FREEZE = 'Administer Freeze / Unfreeze',
+  ADMIN_FREEZE = 'Freeze Business',
```
Comment (severinbeauvais) [inline src/enums/filingNames.ts:2]: Is this filing name used anywhere else where it should be "Unfreeze Business"?

Comment (seeker25) [reply src/enums/filingNames.ts:2]: Could be, we should probably hold off on merging this until Freeze and Unfreeze is working correctly again in DEV (previous to this code merge)

Comment (severinbeauvais) [reply src/enums/filingNames.ts:2]: Then please change this PR to a draft and/or add the Do Not Merge label on it.

Comment (seeker25) [reply src/enums/filingNames.ts:2]: Added, doesn't look like it's used in any other places

**Pull request merged**